AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py

```diff
@@ -1101,32 +1102,35 @@ def test_WindowsAndMac_AssetListSkipOneOfTwoParents_SharedDependencyIsIncluded(s
                                                                                    bundler_batch_helper):
         """
         Test Steps:
-        1. Create Asset List with a parent asset that is skipped
+        1. Create Asset List with multiple parent assets that are skipped
         2. Verify that Asset List was created
         3. Verify that only the expected assets are present in the asset list
         """
         expected_assets = [
-            "testassets/bundlerskiptest_grandparent.dynamicslice",
-            "testassets/bundlerskiptest_parenta.dynamicslice",
-            "testassets/bundlerskiptest_commonchild.dynamicslice"
+            "objects/cube_lod0.azlod",
+            "objects/cube_lod0_index.azbuffer",
+            "resourcepools/defaultvertexbufferpool.pool"
         ]
         # Test Asset Structure:
-        #     Grandparent
-        #      /      \
-        #   ParentA  ParentB
-        #      \      /
-        #     CommonChild
+        #          Grandparent
+        #      /        |        \
+        #   ParentA  ParentB  ParentC
+        #      \        |        /
+        #          CommonChild
 
-        # Even if we exclude "ParentB", we should still have "CommonChild" since it is a product dependency of "ParentA"
+        # Even if we exclude all parents besides "ParentA", we should still have "CommonChild" since it is a product dependency of "ParentA"
         bundler_batch_helper.call_assetLists(
             assetListFile=bundler_batch_helper['asset_info_file_request'],
-            addSeed="testassets/bundlerskiptest_grandparent.dynamicslice",
-            skip="testassets/bundlerskiptest_parentb.dynamicslice"
+            addSeed="objects/cube_lod0.azlod",
+            skip="objects/cube_lod0_position0.azbuffer, objects/cube_lod0_tangent0.azbuffer,"
+                 "objects/cube_lod0_normal0.azbuffer, objects/cube_lod0_bitangent0.azbuffer,"
+                 "objects/cube_lod0_uv0.azbuffer"
         )
         assert os.path.isfile(bundler_batch_helper["asset_info_file_result"])
         assets_in_list = []
         for rel_path in bundler_batch_helper.get_asset_relative_paths(bundler_batch_helper["asset_info_file_result"]):
             assets_in_list.append(rel_path)
+        print(assets_in_list)
```

Comment:
Did you mean to leave this in?

Reply:
No I missed 2 print statements I need to remove